pkgs/compiler-lib/raco/testing.rkt

```diff
@@ -4,7 +4,13 @@
 
 (provide test-log-enabled?
          test-log!
-         test-report)
+         test-report
+         current-test-invocation-directory)
+
+;; records the original "raco test" directory while raco test changes directory
+;; to invoke tests
+(define current-test-invocation-directory
+  (make-parameter #f))
```

Comment:
It’d be ideal if this parameter has a guard that checks the contract and normalizes the path string like `current-directory`.  I see the RackUnit-side change assumes that the parameter returns a *path* as opposed to path string, which makes such change even more important.  Less importantly, a name should be given to `make-parameter`.  (Since you’re changing this file, could you also change `test-log-enabled?` to add a name?)